### PR TITLE
Added cone template (fixes: #330)

### PR DESCRIPTION
**Pull request by cemkod**

### CODE REVIEW

File: elements/templates/a-cone.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<template is="a-template"
```
Comment (cvan) [inline elements/templates/a-cone.html]: this all looks good to me. @jcarpenter, @ngokevin want to take a look?

File: examples/template-cone/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cube</title>
+    <meta name="description" content="Cone - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <a-scene>
+      <a-cone position="0 0 -20" rotation="30 45 30" height="10" radius-top="2" radius-bottom="10" color="#F3BA8D"></a-cone>
```
Comment (cvan) [inline examples/template-cone/index.html]: `35 45 30` looks more "correct" IMO

**Comment by cvan** [comment]
the rest looks great. let me know. thanks!

File: examples/template-cone/index.html (hunk 2 above)
Comment (cemkod) [inline examples/template-cone/index.html]: fixed: https://github.com/cemkod/aframe/commit/a5c92a2aa12db515f387cd089385ac1922ea9060

**Comment by cvan** [comment]
many thanks!

**Pull request merged**